db/new_seeds/scenarios/participants/ects/ect.rb

```diff
@@ -5,7 +5,14 @@ module Scenarios
     module Participants
       module Ects
         class Ect
-          attr_reader :user, :new_user_attributes, :participant_profile, :participant_identity, :teacher_profile, :school_cohort
+          attr_reader :new_user_attributes,
+                      :participant_identity,
+                      :participant_profile,
+                      :school_cohort,
+                      :teacher_profile,
+                      :user
+
+          delegate :ecf_participant_eligibility, :ecf_participant_validation_data, to: :participant_profile
```

Comment:
Wondering if this is necessary?  I'm not sure whether it's a good idea to treat these scenarios as the actual entities or whether we should consider renaming them with a "Builder" suffix e.g. `EctBuilder` or something like that to make it clear. Would be good to get everyones thoughts on that.

Reply:
Agreed. I have removed  all these method and expose only the participant profile created.